src/linear_algebra/affine_space/combination.lean

```diff
@@ -296,6 +296,72 @@ end
 
 end finset
 
+namespace finset
+
+variables (k : Type*) {V : Type*} {P : Type*} [division_ring k] [add_comm_group V] [module k V]
+variables [affine_space V P] {ι : Type*} (s : finset ι)
+
+/-- The weights for the centroid of some points. -/
+@[nolint unused_arguments]
+def centroid_weights (i : ι) := (finset.card s : k) ⁻¹
+
+/-- `centroid_weights` at any point. -/
+@[simp] lemma centroid_weights_apply (i : ι) : s.centroid_weights k i = (finset.card s : k) ⁻¹ :=
+rfl
+
+/-- `centroid_weights` equals a constant function. -/
+lemma centroid_weights_eq_const :
+  s.centroid_weights k = function.const ι ((finset.card s : k) ⁻¹) :=
```

Comment:
Oh, I see you do know about `function.const`. Why not making it the official definition?